lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp

```diff
@@ -447,7 +602,7 @@ LogicalResult LowerClassesPass::processPaths(
       }
 
       // If we are at a leaf, nothing to do.
-      if (std::distance(it->begin(), it->end()) == 0) {
+      if (it->begin() == it->end()) {
```

Comment:
`llvm::size(A, B)` exists specifically to confidently do `std::distance(..)` on ranges, just related tip.

For checking distance of zero, comparison seems good to me! 👍